packages-tests/Comments/CommentRemover/Fixture/another_comment.php.inc

```diff
@@ -1,6 +1,6 @@
 <?php
 
-namespace Rector\Comments\Tests\CommentRemover\Fixture;
+// intentional without namespace
```

Comment:
you can create a class/function and move the content inside a class to ensure no cache conflict phpstan for anonymous function

Reply:
I guess this test uses intentionally no namespace and therefore intentionally does not define a function or class, so it will not define stuff in the global namespace.